src/utilities/api/interactionManager.ts

```diff
@@ -118,6 +118,11 @@ export const updateVoucher = async (id: string, amount: number) =>
     .then((res) => res)
     .catch((err) => err);
 
+export const getMerchantGiftCards = async (seller_id: string, secret: string) =>
+  axios
+    .get(sellers + seller_id + '/gift_cards/' + secret)
```

Comment:
Does it make sense to do some kind of hashing to encrypt the secret before sending over the network? Lmk if you think thats over-engineering  😂

Reply:
In theory maybe we could, but really this isn't the most secure way of doing this at all. It's kind of a "security through obscurity" approach, but the stakes are pretty low here anyway. The server doesn't support anything that fancy yet, so this is the best we've got. I think if vouchers become a more important part of our organization we'll probably implement some kind of more formal authentication system.

Reply:
"security through obscurity" 🤣